bootloaders/riotboot_dfu/main.c

```diff
@@ -27,6 +27,22 @@
 #include "riotboot/slot.h"
 #include "riotboot/usb_dfu.h"
 
+#include "bootloader_selection.h"
+
+#ifdef BTN_BOOTLOADER_PIN
+#include "periph/gpio.h"
+#endif
+
+static bool bootloader_alternative_mode(void)
```

Comment:
Nitpick regarding coding convention
```suggestion
static bool _bootloader_alternative_mode(void)
```